dockers/base-conda/Dockerfile

```diff
@@ -29,7 +29,14 @@ ENV \
     # CUDA_TOOLKIT_ROOT_DIR="/usr/local/cuda" \
     MKL_THREADING_LAYER=GNU
 
-RUN apt-get update -qq --fix-missing && \
+RUN \
+    # https://github.com/NVIDIA/nvidia-docker/issues/1631
+    rm /etc/apt/sources.list.d/cuda.list && \
```

Comment:
CI did not like this :confused:

Reply:
what CI do you mean? I guess it is resolved now :)

Reply:
back then, I was using `wget` and `curl` which aren't installed in the image. I'm now using `apt-key` so the problem should be resolved now.

@carmocca Could you have another look?